<launch>
    <!-- PARAMS -->
    <param name="rgb_image_topic" value="/camera/rgb/image_raw" />
    <!-- <param name="rgb_image_topic" value="/camera/rgb/image_rect_color" /> -->
    <param name="depth_image_topic" value="/camera/depth_registered/image_raw" />
    <param name="camera_info_topic" value="/camera/rgb/camera_info" />
    <param name="camera_fps" value="30" />
    <param name="wanted_fps" value="5" />

    <param name="rgb_save_path" value="/home/RVLuser/data/summit_test/rgb" />
    <param name="rgb_seg_save_path" value="/home/RVLuser/data/summit_test/rgb_seg" />
    <param name="depth_save_path" value="/home/RVLuser/data/summit_test/depth" />
    <param name="depth_seg_save_path" value="/home/RVLuser/data/summit_test/depth_seg" />
    <param name="ply_save_path" value="/home/RVLuser/data/summit_test/ply_seg" />
    <param name="rvl_cfg_path"
        value="/home/RVLuser/rvl-linux/RVLRecognitionDemo_Cupec_DDD2_Detection.cfg" />

    <param name="bag_path" value="/home/RVLuser/data/bags/summit_test_last.bag" />

    <!-- NODES -->

    <param name="load_model" type="bool" value="true" />
    
    <node name="move_robot_home_node" pkg="green_matrix_summit" type="node_move_robot_home.py" output="screen"/>
    <node name="place_environment_node" pkg="green_matrix_summit" type="node_place_environment.py"
        output="screen" />

    <node name="save_images_node" pkg="green_matrix_summit" type="node_save_images_plys.py"
        output="screen" >
        <param name="bag_used" type="bool" value="true" />
        <param name="skip" type="bool" value="true" />
    </node>
    
    <node name="detect_cabinet_model_node" pkg="green_matrix_summit" type="node_detect_cabinet_model.py"
        output="screen" />
    <node name="push_operation_node" pkg="green_matrix_summit" type="node_push_operation.py"
        output="screen" />


    <!-- <arg name="topic_name" default="/build_model" />
    <arg name="message" default="true" />
    <node pkg="rostopic" type="rostopic" name="rostopic_pub_node" output="screen"
        args="pub $(arg topic_name) std_msgs/Bool $(arg message)" /> -->
</launch>